app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
I'd actually say this might be kinda a bad assumption here. If we are changing the data model at all, a lot of existing records might end up with duplicate data in Mongo, and it could go unnoticed because we might only load certain fields, etc...

What I think would be good, for my sanity sake is for you and @colinxfleming to run this through staging or something with a larger data set, and look at records before and after. Just basically check for any gremlins left behind so we know if we need to write any scripts for cleanup, etc... It could very well be that there's nothing to be concerned about, I just get panicky anytime a data model shifts and you use a NoSQL db.